Src/BootCamp.Chapter/ConsoleLogger.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class ConsoleLogger : ILogger
+    {
+        private string _message;
+
+        public ConsoleLogger()
+        {
+        }
+
+        public ConsoleLogger(string message)
+        {
+            _message = message;
+        }
+
+        public void SetMessage(string message)
+        {
+            _message = message;
+        }
+
+        public string GetMessage()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetMessage' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5149394)